pulp-in-one-container.md

```diff
@@ -148,6 +148,21 @@ $ mkdir -p settings pulp_storage pgsql containers run/postgresql
 $ chmod a+w run/postgresql
 ```
 
+If Pulp tasks are stuck in `waiting` status and executing `docker exec -it pulp bash -c 'rq info'` returns 0 workers:
+
+```
+1 queues, 2 jobs total
+
+0 workers, 1 queues
+```
+
+It will be necessary to create a `pulpcore-resource-manager` directory under the `/run` volume:
+
+```console
+$ mkdir -p run/pulpcore-resource-manager
+```
```

Comment:
Will those two problems always occur in conjunction? Then we should just fold this `mkdir` into the above one.

OTOH: `pulpcore-resource-manager` sounds like the working directory of the resource manager worker which should be autocreated. If this fails, i suspect all the other workers to fail to start up too. And there is no way to predict their working directories beforehand. Can you maybe mount the `/run` volume with more permissions?

Reply:
Thanks for the feedback, @mdellweg!

Yes, the problems always occur together (at least from the CentOS7 environment I've been testing)―I've gone back and consolidated them into one section with a single `mkdir`.  

You're also right that the other rq workers fail to start.  I've added a workaround for them, too.  I'm not sure how I can mount the `/run` directory with more permissions; the container already writes the pid files as `root:root`.